tests/ut/language/parser/test_spmd_submit.py

```
@@ -0,0 +1,403 @@
+            def k(
+                self, x: pl.Tensor[[128], pl.FP32], out: pl.Out[pl.Tensor[[128], pl.FP32]]
+            ) -> pl.Tensor[[128], pl.FP32]:
+                t = pl.load(x, [0], [128])
+                out = pl.store(t, [0], out)
+                return out
+
+            @pl.function(type=pl.FunctionType.Orchestration)
+            def main(
+                self, x: pl.Tensor[[128], pl.FP32], out: pl.Out[pl.Tensor[[128], pl.FP32]]
+            ) -> pl.Tensor[[128], pl.FP32]:
+                out, tid = pl.spmd_submit(self.k, x, out, core_num=4, sync_start=True)
```

Ruff (0.15.15)

[warning] Unpacked variable `tid` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)